TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -1775,5 +1776,27 @@ bool laneChangesAllowed
 
             return reverse;
         }
+
+        public bool SupportsTrolleyBus(LaneTransitionData transition, uint otherLaneId) {
+            // should not be invalid.
+            // relaxed is also accepted.
+            if (transition.type == LaneEndTransitionType.Invalid)
```

Comment:
@krzychu124 this is the code that allows relaxed 
it allows all but Invalid so it allows relaxed.
this might not have been obvious because relax is implied but I did explain in the comment :)